web-client/integration-tests/helpers.js

```diff
@@ -927,3 +935,25 @@ export const updateOrderForm = async (cerebralTest, formValues) => {
     'updateAdvancedOrderSearchFormValueSequence',
   );
 };
+
+export const verifySortedRecievedAtDateOfPendingItems = pendingItems => {
```

Comment:
Nit: received

Reply:
Hah!  Nice catch.  This has been fixed here and in the other locations where it was misspelled.